src/components/layout/layout.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Layout: React.FC<Props> = ({ children }: Props) => {
+  return (
+    <>
+      <div className="container mx-auto">{children}</div>;
+    </>
```

Comment:
No need for a [`React.Fragment`](https://reactjs.org/docs/fragments.html#short-syntax), just return the `div` directly

Reply:
You are right. Was playing around with React, forgot to remove.